include/seqan3/argument_parser/detail/version_check.hpp

```diff
@@ -192,7 +192,7 @@ class version_checker
                               " " +
                               out_file.string() +
                               " " +
-                              std::string{"http://seqan-update.informatik.uni-tuebingen.de/check/SeqAn3_"} +
+                              std::string{"http://seqan-update.informatik.uni-tuebingen.de/check/SeqAn_"} +
```

Comment:
Sure ?

Reply:
No !

Reply:
no that's over eagerly replaced.

Reply:
but why did no test catch this? Should there not be at least one failing test? Something, like a dummy app registered on the server?

Reply:
I cannot add a test because if the server does not respond because of some other issue (other than a wrong link) we will have a broken CI that has nothing to do with our library and we wanted to avoid this.

Reply:
Ok, let's say our general assumption is that the server is always online. Would it then not be of help to see that the test is failing. Otherwise, we might not even realize that the server is not responding for whatever reason.

Reply:
What you mean is tested, since I check if given that the server is online and I ping him for the seqan version then I expect a correct result. 
Actually, the change you introduced will NEVER raise a failure because `SeqAn` is just as valid in the URL as `SeqAn3` since we also track the old library version :wink:

Reply:
Well, not if there was a specific app we are expecting to see, right? Like, foo_bar which obviously should not also exist as SeqAn_foo_bar.  But IIRC there were this developer setting. Like if you are a developer and you are not registered there is a response like: Hey don't you want to register and benefit from update notifications? I would assume there is a test that checks for an existing app and would instead get the developer notification, since the server address is not invalid but not as expected?

Reply:
> I would assume there is a test that checks for an existing app and would instead get the developer notification, since the server address is not invalid but not as expected?

I am not sure if I understand you correctly but no, for the developer notification "`please register your app...`" the server call has to be successfull and return `UNKNOWN_APP`. So there will be no message if the server is not available or the call is wrong.

Reply:
> "please register your app..." the server call has to be successfull and return UNKNOWN_APP

But that's what I am referring to. If I change the server address in such a way that it is still valid so the call is successful but there was no app registered with this name because like above SeqAn_foo_bar and SeqAn3_foo_bar are two distinct names. So I assume there is a test that expects the answer of a successful call with a different outcome. Such as there is a new version. So this test would expect to see "NEW_VERSION" but actually gets, "UNKNOWN_APP", so it fails.

Reply:
lets discuss this in person

Reply:
We discussed this offline. so I mark this resolved.